bravado/requests_client.py

```diff
@@ -46,7 +46,7 @@ def matches(self, url):
         :return: True if matches host, port and scheme, False otherwise.
         """
         split = urlparse.urlsplit(url)
-        return self.host == split.hostname
+        return self.host == split.netloc
```

Comment:
Hey @sanda87, thanks for the PR and sorry for the late response! 

While this works in your case, we should also either fix the docstring (we're not checking the scheme), or add that in here as well.

Could you also please add a test to make sure it works as expected, and so that there will be no regressions in the future? Maybe `tests/http_client_test.py` is a good fit for that.

Reply:
Hi @sjaensch , 
I added tests for this case and updated docstring in my new commit